examples/shardingsphere-example-generator/src/main/java/org/apache/shardingsphere/example/generator/core/ExampleGeneratorFactory.java

```diff
@@ -53,13 +54,14 @@ private Configuration createTemplateConfiguration() throws IOException {
     
     /**
      * Generate directories and files by template.
-     * 
+     *
      * @throws TemplateException template exception
      * @throws IOException IO exception
      */
     @SuppressWarnings("unchecked")
     public void generate() throws TemplateException, IOException {
         YamlExampleConfiguration exampleConfiguration = swapConfigToObject();
+        new YamlExampleConfigurationValidator().validate(exampleConfiguration);
```

Comment:
Maybe the tool class is more suitable, we call the static method for verification

Reply:
ok